canvas_manage_course/views.py

```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
 
 import logging
-import urllib
-import urlparse
+import urllib.request, urllib.parse, urllib.error
+import urllib.parse
```

Comment:
Can remove the `import urllib.parse` on line 5 as it's being imported in line 4.